zipkin/pom.xml

```diff
@@ -42,6 +42,13 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
yeah sadly you'll have to do the opposite... explicitly add to collector and storage main poms, not at parent as it leaks here

Reply:
done